packages/cli/src/commands/init/init.js

```diff
@@ -160,16 +225,25 @@ export default (async function initialize(
    */
   const version: string = minimist(process.argv).version || 'latest';
 
-  if (fs.existsSync(projectName)) {
-    throw new DirectoryAlreadyExistsError(projectName);
-  }
+  const {directory, directoryExists} = getProjectDirectory({
+    projectName,
+    directory: options.directory,
+  });
 
   try {
-    await createProject(projectName, options, version);
+    await createProject(
+      projectName,
+      {...options, directory},
```

Comment:
There's a flow error that I haven't been able to fix yet, will investigate it later but I'm open for tips.

![image](https://user-images.githubusercontent.com/6207220/57916255-c0665100-7892-11e9-8628-ea95f077e106.png)